frontend/app/routes/$lang+/_public+/apply+/$id+/confirmation.tsx

```diff
@@ -140,7 +140,7 @@ export default function ApplyFlowConfirm() {
 
   const mscaLink = <InlineLink to={t('confirm.msca-link')} />;
   const dentalContactUsLink = <InlineLink to={t('confirm.dental-link')} />;
-  const cdcpLink = <InlineLink to={t('confirm.cdcp-checker-link')} />;
+  const cdcpLink = <InlineLink to={'/view-application'} />;
```

Comment:
Shouldn't this link go to `/status`?

See https://cdcp-dev.dev-dp.dts-stn.com/en/status